src/site/antora/modules/ROOT/pages/manual/appenders/database.adoc

```diff
@@ -0,0 +1,1505 @@
+
+| useTls
+| `boolean`
+| `true`
+| Whether to use TLS/SSL to connect to Cassandra. This is `false` by default.
+|===
+
+[#CassandraAppender-elements]
+.Cassandra Appender nested elements
+[cols="1m,1,4"]
+|===
+| Type | Multiplicity | Description
+
+| [[CassandraAppender-element-Filter]]xref:manual/filters.adoc[`Filter`]
+| zero or one
+|
+Allows filtering log events just before they are formatted and sent.
+
+See also xref:manual/filters.adoc#appender-stage[appender filtering stage].
+
+| [[CassandraAppender-element-ColumnMapping]]<<ColumnMapping,`ColumnMapping`>>
+| one or more
+a|
+A list of <<ColumnMapping,column mapping>> configurations.
+The following database-specific restrictions apply:
+
+* the <<attr-name,`name` attribute>> must be a valid
+https://cassandra.apache.org/doc/stable/cassandra/cql/definitions.html#identifiers[CQL identifier].
+* the <<attr-literal,`literal` attribute>> must be a valid
+https://cassandra.apache.org/doc/stable/cassandra/cql/definitions.html#terms[CQL term].
+* the <<attr-parameter,`parameter` attribute>> has a fixed value of `?`.
+
+| [[CassandraAppender-element-SocketAddress]]<<SocketAddress,`SocketAddress`>>
+| one or more
+| A list of Cassandra node addresses to connect to.
+If absent, `localhost:9042` will be used.
+
+See <<SocketAddress>> for the configuration syntax.
+|===
+
+Additional runtime dependencies are required for using the Cassandra Appender:
+
+[tabs]
+====
+Maven::
++
+[source,xml]
+----
+<!-- We assume you use `log4j-bom` for dependency management -->
+<dependency>
+  <groupId>org.apache.logging.log4j</groupId>
+  <artifactId>log4j-cassandra</artifactId>
+  <scope>runtime</scope>
+</dependency>
+----
+
+Gradle::
++
+[source,groovy]
+----
+// We assume you use `log4j-bom` for dependency management
+runtimeOnly 'org.apache.logging.log4j:log4j-cassandra'
+----
+
+====
+
+xref:plugin-reference.adoc#org-apache-logging-log4j_log4j-cassandra_org-apache-logging-log4j-cassandra-CassandraAppender[{open-book} Plugin reference for `Cassandra`]
+
+[#SocketAddress]
+=== Socket Addresses
+
+The address of the Cassandra server is specified using the `SocketAddress` element, which supports the following configuration options:
+
+[#SocketAddress-attributes]
+.`SocketAddress` configuration attributes
+[cols="1m,1,1,5"]
+|===
+| Attribute | Type | Default value | Description
+
+| [[SocketAddress-attr-host]]host
+| https://docs.oracle.com/javase/8/docs/api/java/net/InetAddress.html[`InetAddress`]
+| `localhost`
+| The host to connect to.
+
+| [[SocketAddress-attr-port]]port
+| `int`
+| `0`
+| The port to connect to.
+
+|===
+
+xref:plugin-reference.adoc#org-apache-logging-log4j_log4j-core_org-apache-logging-log4j-core-net-SocketAddress[{open-book} Plugin reference for `SocketAddress`]
+
+[#CassandraAppender-examples]
+=== Configuration examples
+
+Here is an example Cassandra Appender configuration:
+
+[source,xml]
+----
+<Configuration name="CassandraAppenderTest">
```

Comment:
This needs to be ported to `examples` in all supported configuration formats.

Reply:
Fixed in https://github.com/apache/logging-log4j2/pull/2790/commits/fad8d7f8603086a8abaa44c0c9e2d1dd6d7524fe